pkg/controller/template/template_controller.go

```diff
@@ -171,12 +182,54 @@ func (ctrl *Controller) deleteSecret(obj interface{}) {
 	}
 }
 
+func (ctrl *Controller) enqueueController() {
+	cfg, err := ctrl.ccLister.Get(ctrlcommon.ControllerConfigName)
+	if err != nil {
+		utilruntime.HandleError(fmt.Errorf("couldn't get ControllerConfig on secret callback %#v", err))
```

Comment:
Missed this one, also needs an update to be non-secret specific here too
```suggestion
		utilruntime.HandleError(fmt.Errorf("couldn't get ControllerConfig on dependency callback %#v", err))
```